# 호핑투어 waypoints
waypoints: [ 
  [37.451133, 126.65612, 50.0], #left bottm
  [37.451115, 126.656213, 49.0], #mid up
  [35.069481592976622, 128.57889197025276, 49.0],
  [35.069464319011999, 128.57896821820279, 49.0], #right bottom 
  [35.069558500130611, 128.57893036813357, 49.0], #right up
  [35.069387336800794, 128.57896450335267, 49.0] # startpoint
]


# 도착 인정 범위 (meters)
goal_range: 1.5
# 각도-서보모터 PID 계수
kp_servo : 0.4
ki_servo : 0.0
kd_servo : 0.1

# 거리-선속 PID 계수
kp_distance : 10
ki_distance : 0
kd_distance : 0

servo_range: [78, 113]
# 쓰러스터 PWM 신호 (선속) (1500 정지 ~ 최대 1900)
thruster: 1600
thruster_max : 1800
thruster_min : 1750

######################### 2021 Version #########################

# ENU 변환 원점
# map_dd : {
#   map_00_lat : 35.069387336800794, 
#   map_00_lon : 128.57896450335267,
#   map_00_alt : 49.0
#   }
